dial-docker-compose/ci/addon/azure_deployment_mock/app.py

```diff
@@ -0,0 +1,44 @@
+import json
+import os
+from fastapi import FastAPI, Request, HTTPException
+from fastapi.responses import StreamingResponse
+from responses import responses, create_response
+
+app = FastAPI()
+
+API_KEY = os.environ.get("API_KEY")
+
+if API_KEY is None:
+    raise ValueError("API_KEY environment variable not set")
+
+response_idx = 0
+
+
+@app.post("/openai/deployments/{model}/chat/completions")
+async def process_completion(model: str, request: Request):
```

Comment:
Dial sdk could help simplify the code, although not sure if needed.

Reply:
Maybe, but I don't want to bring DIAL-specific stuff here, it seems backwards.